ui/tests/acceptance/config-ui/login-settings/login-settings-test.js

```diff
@@ -6,36 +6,105 @@
-      return {
-        data: {
-          key_info: 'key_info',
-          keys: ['key1', 'key2'],
+  hooks.beforeEach(function () {
+    syncScenario(this.server);
+    syncHandlers(this.server);
+    this.server.post('/sys/capabilities-self', allowAllCapabilitiesStub());
+    const listPayload = {
+      keys: ['ns1', 'ns2'],
+      keyInfo: {
+        ns1: {
+          DisableInheritance: true,
+          Name: 'ns1',
+          Namespace: 'root',
+        },
+        ns2: {
+          DisableInheritance: true,
+          Name: 'ns2',
+          Namespace: 'root',
         },
-      };
-    });
+      },
+    };
+
+    const rulePayload = {
+      data: {
+        backup_auth_types: ['okta', 'userpass'],
+        default_auth_type: 'ldap',
+        disable_inheritance: true,
+        namespace: 'root/',
+      },
+    };
+
+    const api = this.owner.lookup('service:api');
+    this.ruleListStub = sinon.stub(api.sys, 'uiLoginDefaultAuthList').resolves(listPayload);
+    this.singleRuleStub = sinon.stub(api.sys, 'uiLoginDefaultAuthReadConfiguration').resolves(rulePayload);
 
-    // Visit the login settings index page
-    await visit('/login-settings');
+    return login();
+  });
+
+  test('fetched login rule list renders', async function (assert) {
+    // Visit the login settings list index page
+    await visit('vault/config-ui/login-settings');
 
-    // Check if the page is rendered correctly
-    assert.strictEqual(currentURL(), '/login-settings', 'Navigated to login settings index page');
+    // verify fetched rules are rendered in list
+    assert.dom('.linked-block-item').exists({ count: 2 });
   });
 
-  // keep a render test
+  test('delete rule from list view', async function (assert) {
+    // Visit the login settings list index page
+    await visit('vault/config-ui/login-settings');
 
-  // fetched login rules list should display on index page
+    await click(GENERAL.menuTrigger);
+    await click(GENERAL.menuItem('delete-rule'));
+    await click(GENERAL.confirmButton);
+
+    // verify success message from deletion
+    assert.dom(GENERAL.latestFlashContent).includesText('Successfully deleted rule ns1.');
+  });
 
-  // clicking a rule should navigate to the details page (verify route & render)
+  test('navigate to rule details page and renders rule data', async function (assert) {
+    // visit individual rule page
+    await visit('vault/config-ui/login-settings/');
 
-  // from login list index page, clicking delete should have a popup modal, should be removed from list on confirm
+    await click(GENERAL.menuTrigger);
+    await click(GENERAL.menuItem('view-rule'));
+
+    // verify that user is redirected to the rule details page
+    assert.strictEqual(
+      currentRouteName(),
+      'vault.cluster.config-ui.login-settings.rule.details',
+      'goes to rule details page'
+    );
+
+    // verify fetched rule data is rendered
+    assert.dom('.info-table-row').exists({ count: 5 });
```

Comment:
For this, can you use [the selectors](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L66) instead and maybe verify at least the value?